tools/make_patch.c

```diff
@@ -145,7 +157,7 @@ int parse_address(char *buffer)
 	}
 
 	// If it's not a ROM address, (silently) ignore it.
-	if (address > 0x7FFF) return 0;
+	//if (address > 0x7FFF) return 0;    <-- This was filtering out hJoyPressed and a lot of other stuff. Turned off for now...
```

Comment:
Just remove the code. Don't leave it commented. We get the reasoning.

Reply:
.... This was already resolved in 6eeec39b896a70a29c740b709174f0e17b57540c